src/scheduler.rs

```diff
@@ -122,7 +122,7 @@ async fn fetch_idle(ctx: &Context, connection: &mut Imap, folder: Config) -> Int
         Some(watch_folder) => {
             // connect and fake idle if unable to connect
             if let Err(err) = connection.connect_configured(&ctx).await {
-                error!(ctx, "imap connection failed: {}", err);
+                warn!(ctx, "imap connection failed: {}", err);
```

Comment:
doesn't this loose all error messaging to the user?  i.e. what happens with authentication failures during configure?

Reply:
This code shouldn't even be called until `start_io`, which starts the scheduler.